v5/releases/cli3.1.4.rst

```diff
@@ -29,7 +29,9 @@ Download and run the appropriate installer
 macOS
 -----
 
-See below for the new recommended method of installing the CLI
+To upgrade a previous installation, run ``brew upgrade pros-cli``
+
+To install from scratch:
 
 macOS installs
 ~~~~~~~~~~~~~~
```

Comment:
still need to delete this line (35) and the one above it. I'd make a suggestion but i'm honestly not sure how to do multiline suggestions like that